pkg/commands/git_commands/repo_paths_test.go

```diff
@@ -1,30 +1,41 @@
 package git_commands
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/go-errors/errors"
-	"github.com/spf13/afero"
+	"github.com/jesseduffield/lazygit/pkg/commands/oscommands"
 	"github.com/stretchr/testify/assert"
 )
 
-func mockResolveSymlinkFn(p string) (string, error) { return p, nil }
-
 type Scenario struct {
 	Name       string
-	BeforeFunc func(fs afero.Fs)
+	BeforeFunc func(runner *oscommands.FakeCmdObjRunner)
 	Path       string
 	Expected   *RepoPaths
 	Err        error
 }
 
-func TestGetRepoPathsAux(t *testing.T) {
+func TestGetRepoPaths(t *testing.T) {
```

Comment:
Sorry for not mentioning this in my first review: now that GetRepoPaths is just directly pulling the values from `git rev-parse`, we can scrap some of these unit tests. We only really need three tests here:
1) when we're in a submodule
2) when we're not in a submodule
3) when git rev-parse returns an error

That covers all the conditional logic in our code.